apps/shade/src/components/layout/heading.tsx

```diff
@@ -0,0 +1,60 @@
+import {cn} from '@/lib/utils';
```

Comment:
Oh I see the prefix is used here - this is a new pattern, am interested in discussing it further!